### DISCUSSION TITLE
Code Quotes

### DISCUSSION THREAD

**Discussion by lzcunt** [monkeytypegame/monkeytype#798] [Ideas] [answered]
So I have a great idea, **CODE QUOTES!**

# How it works
Quotes get non-English language support, when selecting code languages, it gives random lines from MonkeyType's (and other popular open source software's) source code.

So if I select Quote Code JS, I might get: 
```js
if (!document.querySelector("#monkey").classList.contains("hidden")) {
```

As a side feature, non-English real world language quotes could be added.

**Comment by Miodec** [comment, marked as answer]
Just added multi language quote support. Will work on a way to submit quotes, but you should be able to submit some if you want to, by creating a new json file inside the quotes folder.
